parser/model/ddl.go

```diff
@@ -822,27 +824,52 @@ func (job *Job) IsRollbackable() bool {
 	return true
 }
 
+func (job *Job) IsPausing() bool {
+	return job.State == JobStatePausing
+}
+func (job *Job) IsPaused() bool {
+	return job.State == JobStatePaused
+}
+func (job *Job) IsResuming() bool {
+	return job.IsPaused()
+}
+
+func (job *Job) IsPausable() bool {
+	return (job.NotStarted() || job.IsRunning()) && job.IsRollbackable()
+}
+func (job *Job) IsResumable() bool {
+	return job.IsPaused()
+}
+
 // JobState is for job state.
-type JobState byte
+type JobState int32
```

Comment:
Why change jobState to int32?  IMO byte could store total 256 states.

Reply:
atomic change to notify workers atomicly.